library-best-practices/JLBP-17.md

```diff
@@ -0,0 +1,61 @@
+[JLBP-17] Coordinate Major Version Adoption
+-------------------------------------------
+
+For any particular library dependency chain, the libraries involved need to
+coordinate the adoption of new major versions of any shared dependency that has
+breaking changes. This is necessary so that consumers can easily use the latest
+version of all of the functionality together.  This best practice doesn't apply
+to the adoption of new major versions where the package was renamed, because the
+old and new packages can co-exist.
+
+As an example of what happens when this is not followed, imagine the partial
```

Comment:
we can drop "imagine the partial
adoption of a new major version of a leaf library." since you cover this and two other cases in sequence here.

Reply:
I think it helps to know what's coming though, so you know why it's relevant information.